bedrock/base/templates/base-pebbles.html

```diff
@@ -102,9 +102,15 @@
             <li class="first about-item">
               <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{_('About')}}</a>
             </li>
+            {% if LANG.startswith('en-') %}
+            <li class="technology-item">
+              <a href="{{ url('mozorg.technology') }}" data-link-type="nav" data-link-name="Web Innovations" data-alt-text="{{ _('Innovations') }}">{{ _('Web Innovations') }}</a>
```

Comment:
How is the data-link-name attribute used in practice? All other menus only have one string, to avoid confusion I would personally stick to that